code/modules/clothing/spacesuits/void/military_vr.dm

```diff
@@ -15,13 +15,18 @@
 
 /obj/item/clothing/head/helmet/space/void/merc/prototype
```

Comment:
Rather than override the stats, it would be better to just re-path it into a security voidsuit subtype.

Reply:
Done.